Formula/telepresence.rb

```diff
@@ -0,0 +1,24 @@
+class Telepresence < Formula
+  desc "Fast, efficient local development for Kubernetes microservices"
+  homepage "https://www.telepresence.io"
+  url "https://github.com/telepresenceio/telepresence.git",
+      tag:      "v2.8.2",
+      revision: "58ddcc58e5d5cb5ee15e159b26b6399400ebbf5c"
+  license "Apache-2.0"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "build"
+    bin.install "build-output/bin/telepresence"
+  end
+
+  test do
+    system "#{bin}/telepresence", "status"
```

Comment:
Can we check some output here?

Reply:
```
❯ telepresence status
Root Daemon: Not running
User Daemon: Not running
```
Any tips how should I check multiline output, I'm barely worked with Ruby.